pages/events/event_register.php

```diff
@@ -1,13 +1,25 @@
 <?php
 restrict_access();
 
+$id = $_SESSION['user_id'];
+
 require_once "database/events.api.php";
 $event = get_event_by_id(get_route_param('event_id'), $_SESSION['user_id']);
 $competitions = get_competitions_by_event_id($event['did'], $_SESSION['user_id']);
 
+$competitions_id = [];
```

Comment:
this is probably not needed. You can name your fields in a way that should be readable and understandable for both humans and logic. What you can do instead:
- have a dynamic name for the fields with the competition ids in them (not recommended, it will make it harder to code the validation)
- use `name=something[]`